### PR TITLE
Doc 2894 bug bash   rc security pt2

### PR DESCRIPTION
**Pull request by kurtfm**
https://docs.redis.com/staging/DOC-2894-Bug-Bash---RC-Security-pt2/rc/security/access-control/data-access-control/create-assign-users/

- clear up each step (mainly because I added screenshots)
- add screenshots

includes: DOC-2892

### CODE REVIEW

**Comment by kurtfm** [comment]
Still waiting for one image to update in staging... caching thing.  Should be ready to review.

File: content/rc/security/access-control/data-access-control/configure-acls.md [hunk 1]
```diff
@@ -22,13 +22,27 @@ ACLs that are not marked with the Redis logo are user-defined ACL rules.
 
 To configure a Redis ACL that you can assign to a data access role:
 
-1. Go to **Data Access Control > ACLs** and either select `+` to create a new Redis ACL or point to an existing ACL and select the pencil icon to edit it.
+1. Go to **Data Access Control**
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/configure-acls.md:25]:
```suggestion
1. Go to **Data Access Control** from the [Redis Cloud console](https://app.redislabs.com/#/) menu.
```

File: content/rc/security/access-control/data-access-control/configure-acls.md [hunk 2]
```diff
@@ -22,13 +22,27 @@ ACLs that are not marked with the Redis logo are user-defined ACL rules.
...
 
-1. Provide a descriptive name for the Redis ACL.
+    {{<image filename="images/rc/data-access-control-menu.png" width="200px" alt="Menu for database access control." >}}{{< /image >}}
 
-1. Create the ACL rule. Enter the rule [using ACL syntax](#define-permissions-with-acl-syntax).
+1. Select the **Redis ACLs** tab
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/configure-acls.md:29]:
```suggestion
1. Select the **Redis ACLs** tab.
```

File: content/rc/security/access-control/data-access-control/configure-acls.md [hunk 3]
```diff
@@ -22,13 +22,27 @@ ACLs that are not marked with the Redis logo are user-defined ACL rules.
...
 
-1. Select the check mark to save your changes.
+    {{<image filename="images/rc/data-access-control-redis-acls.png" alt="Redis ACLs area." >}}{{< /image >}}
+
+1. Either select `+` to create a new Redis ACL or point to an existing ACL and select the pencil icon to edit it.
+
+    {{<image filename="images/rc/data-access-control-redis-acls-add-or-update.png" width="400px" alt="Add or Update Redis ACL." >}}{{< /image >}}
+
+    {{< note >}}Some of the built-in Redis ACLs are not editable.{{< /note >}}
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/configure-acls.md:37]:
```suggestion
    {{< note >}}The built-in Redis ACLs can't be edited.{{< /note >}}
```
Unless there's a built-in ACL that can be edited, in which case:

```suggestion
    {{< note >}}Some of the built-in Redis ACLs can't be edited.{{< /note >}}
```

File: content/rc/security/access-control/data-access-control/create-assign-users.md [hunk 4]
```diff
@@ -16,30 +16,57 @@ Before you create a database user, you must [create a data access role]({{<relre
 
 To create a user:
 
-1. Go to **Data Access Control > Users**.
+1. Go to **Data Access Control**
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/create-assign-users.md:19]:
```suggestion
1. Go to **Data Access Control** from the [Redis Cloud console](https://app.redislabs.com/#/) menu.
```

File: content/rc/security/access-control/data-access-control/create-assign-users.md [hunk 5]
```diff
@@ -16,30 +16,57 @@ Before you create a database user, you must [create a data access role]({{<relre
...
 
-1. Select the `+` button to create a new user.
+    {{<image filename="images/rc/data-access-control-menu.png" width="200px" alt="Menu for database access control." >}}{{< /image >}}
 
-1. Enter a username.
+1. Select the **Users** tab
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/create-assign-users.md:23]:
```suggestion
1. Select the **Users** tab.
```

File: content/rc/security/access-control/data-access-control/create-assign-users.md [hunk 6]
```diff
@@ -16,30 +16,57 @@ Before you create a database user, you must [create a data access role]({{<relre
...
+
+    {{<image filename="images/rc/data-access-control-users-no-users.png" alt="User configuration area." >}}{{< /image >}}
+
+2. Select `+` to create a new user
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/create-assign-users.md:27]:
```suggestion
2. Select `+` to create a new user.
```

File: content/rc/security/access-control/data-access-control/create-assign-users.md [hunk 7]
```diff
@@ -16,30 +16,57 @@ Before you create a database user, you must [create a data access role]({{<relre
...
+
+    {{<image filename="images/rc/data-access-control-users-add-or-edit.png" width="300px" alt="User add or edit." >}}{{< /image >}}
+
+3. Enter a username.
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/create-assign-users.md:31]:
```suggestion
3. Enter a username in the **Username** field.
```

File: content/rc/security/access-control/data-access-control/create-assign-users.md [hunk 8]
```diff
@@ -16,30 +16,57 @@ Before you create a database user, you must [create a data access role]({{<relre
...
     {{<note>}}
 An error occurs if a user tries to connect to a memcached database with the username `admin`. Do not use `admin` for a username if the user will be connecting to a memcached database.
     {{</note>}}
 
 1. Select a [**Role**]({{<relref "rc/security/access-control/data-access-control/create-roles">}}) from the list.
 
-1. Enter and confirm the user's password.
+    {{<image filename="images/rc/data-access-control-users-add-role.png" width="300px" alt="User select role." >}}{{< /image >}}
+
+1. Enter and confirm the user's password then select the check mark to save the user.
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/create-assign-users.md:43]:
```suggestion
1. Enter and confirm the user's password, then select the check mark to save the user. 
```

File: content/rc/security/access-control/data-access-control/create-assign-users.md [hunk 9]
```diff
@@ -16,30 +16,57 @@ Before you create a database user, you must [create a data access role]({{<relre
...
+    {{<image filename="images/rc/data-access-control-users-password-and-finish.png" width="300px" alt="User add password and finish." >}}{{< /image >}}
 
-1. Select the check mark to save the user.
 
 ## Assign roles to existing users
 
 To assign a data access role to an existing user:
 
-1. Go to **Data Access Control > Users**.
+1. Go to **Data Access Control**
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/create-assign-users.md:52]:
```suggestion
1. Go to **Data Access Control** from the [Redis Cloud console](https://app.redislabs.com/#/) menu.
```

File: content/rc/security/access-control/data-access-control/create-assign-users.md [hunk 10]
```diff
@@ -16,30 +16,57 @@ Before you create a database user, you must [create a data access role]({{<relre
...
+
+    {{<image filename="images/rc/data-access-control-menu.png" width="200px" alt="Menu for database access control." >}}{{< /image >}}
+
+1. Select the **Users** tab
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/create-assign-users.md:56]:
```suggestion
1. Select the **Users** tab.
```

File: content/rc/security/access-control/data-access-control/create-roles.md [hunk 11]
```diff
@@ -12,15 +12,29 @@ aliases:
 
 To assign [Redis ACLs]({{<relref "rc/security/access-control/data-access-control/configure-acls">}}) to a data access role:
 
-1. Go to **Data Access Control > Roles** and either select `+` to create a new role or point to an existing role and select the pencil icon to edit it.
+1. Go to **Data Access Control**
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/create-roles.md:15]:
```suggestion
1. Go to **Data Access Control** from the [Redis Cloud console](https://app.redislabs.com/#/) menu.
```

File: content/rc/security/access-control/data-access-control/create-roles.md [hunk 12]
```diff
@@ -12,15 +12,29 @@ aliases:
...
 
-1. In the **Associations** section of the **Edit role** or **Create new role** screen, you can select `+` to create a new association or point to an existing association and select the pencil icon to edit it.
+    {{<image filename="images/rc/data-access-control-menu.png" width="200px" alt="Menu for database access control." >}}{{< /image >}}
 
-1. Select one or more databases from the **Databases** list.
+1. Select the **Roles** tab
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/create-roles.md:19]:
```suggestion
1. Select the **Roles** tab.
```

File: content/rc/security/access-control/data-access-control/create-roles.md [hunk 13]
```diff
@@ -12,15 +12,29 @@ aliases:
...
-1. Select the check mark to confirm the association.
+1. Select `+` to create a new role or point to an existing role and select the pencil icon to edit it.
+
+    {{<image filename="images/rc/data-access-control-roles-add-or-edit.png" width="300px" alt="Add or edit a role." >}}{{< /image >}}
+
+1. Enter a name for the role.
+
+    {{<image filename="images/rc/data-access-control-roles-add.png" width="400px" alt="Role add screen." >}}{{< /image >}}
+
+1. Select a **ACL rule**
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/create-roles.md:31]:
```suggestion
1. Select an **ACL rule** to assign to the role.
```

File: content/rc/security/access-control/data-access-control/role-based-access-control.md [hunk 14]
```diff
@@ -35,6 +35,10 @@ Finally, in the **Users** tab, you [create users]({{<relref "rc/security/access-
 
 {{<image filename="images/rc/data-access-control-users.png" alt="Data access control screen." >}}{{< /image >}}
 
+{{<note>}}
+Users created for data access are different from the account access users.{{</note>}}
```
Comment (cmilesb) [inline content/rc/security/access-control/data-access-control/role-based-access-control.md:39]: I would add a link here to the access management docs.

```suggestion
Database access users are different from account access users. To learn more, see [Access management](INSERT RELREF HERE).{{</note>}}
```

**Review by cmilesb — CHANGES_REQUESTED** [review]
Just a few things.

File: content/rc/security/access-control/data-access-control/role-based-access-control.md (hunk 14 above)
Comment (kurtfm) [reply content/rc/security/access-control/data-access-control/role-based-access-control.md:39]: I will fix this directly

**Comment by kurtfm** [comment]
@cmilesb I committed all your suggestions except the note change which I did and pushed direclty

**Review by cmilesb — APPROVED** [review]

**Pull request merged**